lib/kitchen/driver/openstack.rb

```diff
@@ -303,14 +302,22 @@ def get_public_private_ips(server)
       end
 
       def get_ip(server)
-        unless config[:floating_ip].nil?
+        if config[:floating_ip]
           debug "Using floating ip: #{config[:floating_ip]}"
           return config[:floating_ip]
         end
+
+        # make sure we have the latest info
+        info 'Waiting for network information to be available...'
+        w = server.wait_for { reload && !addresses.empty? }
```

Comment:
Yeah i'm good with the repurpose `winrm_wait` as a general wait. We'll need to update the README and all to reflect it, but no point in having two things that in essence do the exact same thing.